### PR TITLE
productRoutes

### PR DESCRIPTION
**Pull request by cneha88**
api/products#16

### CODE REVIEW

File: server/api/product.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }]
+  })
+    .then(products => res.send(products))
+    .catch(next)
+})
+
+
+router.get('/:productId', function (req, res, next) {
+  Product.findAll({
+    where: { productid: req.params.productId },
```
Comment (satsivi) [inline server/api/product.js]: You use req.params.productId here but also put product on req.product. So you can use req.product.id or take out the router.param section above and use req.params.productId

File: server/api/product.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    include: [{ model: Review }]
+  })
+    .then(products => res.send(products))
+    .catch(next)
+})
+
+
+router.delete('/:productId', function (req, res, next) {
+  Product.destroy({ where: { id: req.params.productId } })
```
Comment (satsivi) [inline server/api/product.js]: Same thing here with req.products.id vs req.params.productId

File: server/api/product.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    .then(() => res.sendStatus(204))
+    .catch(next)
+})
+
+router.put('/:productId', function (req, res, next) {
+  Product.findById(req.params.id)
```
Comment (satsivi) [inline server/api/product.js]: Same thing here with req.products.id vs req.params.productId

File: server/api/product.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    .then(function (product) {
```
Comment (satsivi) [inline server/api/product.js]: You used es6 earlier on, so here you could also use it by doing

.then(product => product.update(req.body)
.then(updatedProduct => res.send(updatedProduct)
.catch(next)

File: server/api/index.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 const router = require('express').Router()
 module.exports = router
 
-router.use('/users', require('./users'))
+router.use('/users', require('./users'));
+router.use('/product', require('./product'));
```
Comment (vsmith214) [inline server/api/index.js]: think product should be plural

**Pull request merged**